provider/pkg/runner/sshcommand.go

```diff
@@ -18,14 +18,16 @@ type SSHCommand struct {
 	command     string
 	environment map[string]string
 	payload     []FileAsset
+	keepPayload *bool
```

Comment:
I think you're better off just exposing the regular `RunnerConfig` that we're already using.  I'm already thinking of places where things like the apt lock are going to be necessary (to work around parallel scripts running apt commands at the same time), so we might as not reinvent the wheel.